src/coreclr/src/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -436,6 +436,13 @@ private void Get_CORINFO_SIG_INFO(MethodDesc method, CORINFO_SIG_INFO* sig, bool
         {
             Get_CORINFO_SIG_INFO(method.Signature, sig);
 
+#if READYTORUN
```

Comment:
Nit: This ifdef should not be need. `RhpGcPoll` helper that is needed for this in CoreRT should be a very boring piece of boiler plate code.

Reply:
True. I guess it's better to get a break at the time this is integrated than to forget to turn this on later.